openwrt/configs/mt7622-generic.config

```diff
@@ -0,0 +1,25 @@
+# CONFIG_PACKAGE_kmod-ata-ahci-mtk is not set
```

Comment:
Thanks for marking all the `not set` features. This really helps somebody unfamiliar with all the bells and whistles like me understand what's going on.